pkg/collector/corechecks/network-devices/cisco-sdwan/payload/devices.go

```diff
@@ -32,6 +34,7 @@ func GetDevicesTags(namespace string, devices []client.Device) map[string][]stri
 	deviceTags := make(map[string][]string)
 	for _, device := range devices {
 		deviceTags[device.SystemIP] = buildDeviceTags(namespace, device)
+		deviceTags[device.SystemIP] = append(deviceTags[device.SystemIP], fmt.Sprintf("%s:%s", deviceUserTagResourcePrefix, buildDeviceID(namespace, device)))
```

Comment:
I think this will also end up adding this tag to remote devices (by prefixing it by `remote_`) [here](https://github.com/DataDog/datadog-agent/blob/33facf6aa6f93bd8970d2630ab8d35f8b81f9776/pkg/collector/corechecks/network-devices/cisco-sdwan/report/metrics.go#L385-L387)